### PR TITLE
fixes #53 summary bug

### LINKED ISSUE #53: `--summary` not working
`--summary` flag is not working for me any more on latest version - this bug is reproduced in latest builds on Travis.  Last build which is displaying summary is https://travis-ci.org/haf/expecto/jobs/188807194 (for 6fd15ac33d1cb5d42e766d613e710e7bc53de91c), next one is not displaying summary any more (https://travis-ci.org/haf/expecto/jobs/188808798).... But it doesn't make any sense for me since dd99ef503f02b0ddf25ea55e825b7332f8bc1e86 (commit for which build is not displaying summary anymore) was just version bump, without any changes in code.

@haf, any ideas?

### PR DESCRIPTION
**Pull request by AnthonyLloyd**
@haf  @Krzysztof-Cieslak 

Fixes problems with me stomping all over the config.

The summary still didn't come through. I had to change logger.info to logger.logWithAck.

### CODE REVIEW

**Comment by Krzysztof-Cieslak** [comment]
Working on Travis 🎉

File: Expecto/Expecto.fs [hunk 1]
```diff
@@ -334,7 +334,21 @@ module Impl =
       errored  = get (TestResult.Error null)
       duration = results |> Seq.map (fun r -> r.duration) |> Seq.fold (+) TimeSpan.Zero }
 
-  let logSummary (summary : TestResultSummary) =
+  type Log =
+    {
+        Async: bool ref
```
Comment (haf) [inline Expecto/Expecto.fs]: Properties should be lower-case.

**Comment by haf** [comment]
I'm sorry, but this is a horrible hack.

**Comment by haf** [comment]
v2.4.2 released

**Comment by haf** [comment]
Also, please read the DEVGUIDE so I don't have to keep renaming the properties, methods and types. Much appreciated.

**Comment by Krzysztof-Cieslak** [comment]
@haf, maybe we could set up this as a rule in FSharpLint and make linting part of the build process?

**Comment by AnthonyLloyd** [comment]
Yes its not ideal. I have a better way to do this that I will PR later.

Will continue to work on the styling guide. Its quite different to mine.

**Pull request merged**